commit 7b736592c14d1e7ec0b2f035849d7cecf34d5f25
Author: Mike Frysinger <vapier@gentoo.org>
Date:   Wed Apr 11 16:01:29 2012 -0400

    mmap64: include stdint.h for uint64_t
    
    Some targets include this implicitly, but pull it in explicitly for those
    which don't to fix building for them.
    
    Signed-off-by: Mike Frysinger <vapier@gentoo.org>

diff --git a/libc/sysdeps/linux/common/mmap64.c b/libc/sysdeps/linux/common/mmap64.c
index 3c97c84..cebf961 100644
--- a/libc/sysdeps/linux/common/mmap64.c
+++ b/libc/sysdeps/linux/common/mmap64.c
@@ -10,6 +10,7 @@
 #ifdef __UCLIBC_HAS_LFS__
 
 #include <errno.h>
+#include <stdint.h>
 #include <unistd.h>
 #include <sys/mman.h>
 #include <sys/syscall.h>
